Add Cloud Hosted Release Notes book

lcawl opened the pull request:
This PR is intended as an alternative to https://github.com/elastic/docs/pull/3201, which aims to continue publishing the old Cloud Hosted release notes in the old docs system.

shainaraskas commented:
I'm not sure if this is on the cloud side or this side, but I think something is wrong here because we're getting all of the pages rendered on one big super-page, no sidebar, and links are not working

e.g. https://docs_bk_3202.docs-preview.app.elstc.co/guide/en/cloud-hosted/current/index.html

<img width="1964" alt="image" src="https://github.com/user-attachments/assets/d2eac3ca-04ba-4101-81f1-f762716b2d4c" />

lcawl commented:
> I'm not sure if this is on the cloud side or this side, but I think something is wrong here because we're getting all of the pages rendered on one big super-page, no sidebar, and links are not working

Ah, it worked locally so I think it's because I used the "single: 1" which makes sense in the TPEC case but not here.

shainaraskas commented:
couple small things: 

* variables are not rendering:

		<img width="730" alt="image" src="https://github.com/user-attachments/assets/29f4fca2-b42d-4115-a780-c2fa5ac756e1" />

* title should be changed: 

       <img width="823" alt="image" src="https://github.com/user-attachments/assets/aae33024-7cbe-4e39-9fe6-c789d3689be3" />
       
* "Elasticsearch Service" should be replaced with "Elastic Cloud Hosted" everywhere (or maybe "Elastic Cloud" given that we also have console updates in these release notes?) 

otherwise, the structure looks good to me. we do have a lot of duplicate headings in the sidebar, but those were present in the old system.

I am a little concerned about the links though - are we cool to rely on redirects to get people to the right place? e.g.:
<img width="758" alt="image" src="https://github.com/user-attachments/assets/ce3e3770-6b68-402a-a5a0-5365aadb3dfe" />
https://docs_bk_3202.docs-preview.app.elstc.co/guide/en/cloud-hosted/current/ec-release-notes-2019-10-23.html

lcawl commented:
> couple small things:
> 
> * variables are not rendering:

I've got this working by adding references to the version and shared attribute files in https://github.com/elastic/cloud/pull/139928

> * title should be changed:

I changed it to "Elastic Cloud Release Notes" in https://github.com/elastic/cloud/pull/139928 but if there's a better name that's fine by me.

> * "Elasticsearch Service" should be replaced with "Elastic Cloud Hosted" everywhere (or maybe "Elastic Cloud" given that we also have console updates in these release notes?)

I updated the `n` attribute in https://github.com/elastic/cloud/pull/139928

> I am a little concerned about the links though - are we cool to rely on redirects to get people to the right place?

I think since these are old release notes it's okay to have them point to the old docs. They're now working (i.e. pointing to "current") in https://github.com/elastic/cloud/pull/139928

shainaraskas reviewed (approved):
as long as this looks right after https://github.com/elastic/cloud/pull/139928, that all works for me !

lcawl commented:
run docs-build
